File: exercises/beecrowd/1012/1012.go
// https://judge.beecrowd.com/pt/problems/view/1012
package main

import (
	"fmt"
	"math"
)

func main() {
	const PI float64 = 3.14159
	var a, b, c float64

	fmt.Scanf("%f %f %f", &a, &b, &c)

	fmt.Printf("TRIANGULO: %.3f\n", a*c/2)
	fmt.Printf("CIRCULO: %.3f\n", PI*math.Pow(c, 2))
	fmt.Printf("TRAPEZIO: %.3f\n", (a+b)*c/2)
	fmt.Printf("QUADRADO: %.3f\n", math.Pow(b, 2))
	fmt.Printf("RETANGULO: %.3f\n", a*b)
}

File: exercises/beecrowd/1012/1012.ts
// https://judge.beecrowd.com/pt/problems/view/1012
import * as fs from 'fs';

function main(): void {
    const input = fs.readFileSync('/dev/stdin', 'utf8');
    const lines = input.split(/\r?\n/);
    
    const PI = 3.14159
    const line = String(lines.shift()).split(' ');
    
    const a = Number(line[0]);
    const b = Number(line[1]);
    const c = Number(line[2]);
    
    console.log(`TRIANGULO: ${(a*c/2).toFixed(3)}`)
    console.log(`CIRCULO: ${(PI*Math.pow(c, 2)).toFixed(3)}`)
    console.log(`TRAPEZIO: ${((a+b)*c/2).toFixed(3)}`)
    console.log(`QUADRADO: ${(Math.pow(b, 2)).toFixed(3)}`)
    console.log(`RETANGULO: ${(a*b).toFixed(3)}`)
}

main()